napari/util/status_messages.py

```diff
@@ -1,4 +1,4 @@
-from collections import Iterable
+from collections.abc import Iterable
```

Comment:
what's happening here? seems to be fine, but just curious

Reply:
On 3.7 it throws a warning that the import has been moved and will stop working in 3.8.